src/cider-velox/benchmark/expression/CiderComparisonConjunct.cpp

```diff
@@ -181,6 +181,14 @@ class ComparisonBenchmark : public functions::test::FunctionBenchmarkBase {
 
 std::unique_ptr<ComparisonBenchmark> benchmark;
 
+#define BENCHMARK_GROUP(name, expr)                                                    \
+  BENCHMARK(name##Velox) { benchmark->runVelox(expr); }                                \
+  BENCHMARK_RELATIVE(name##Cider) { benchmark->runCiderCompute(expr); }                \
+  BENCHMARK_RELATIVE(name##Nextgen) { benchmark->runCiderCompute(expr, true); }        \
+  BENCHMARK(name##CiderCompile) { benchmark->runCiderCompile(expr); }                  \
+  BENCHMARK_RELATIVE(name##NextgenCompile) { benchmark->runCiderCompile(expr, true); } \
```

Comment:
Why not also change Comparison cases to BatchProcessor API?

Reply:
we don't need the comparison， just put comparison op in project expr， like how velox is doing （select a>b）.

Reply:
Then okay to remove this file if it's already covered in project？

Reply:
ok to remove

Reply:
Then better to rename `CiderSimpleArithmetic.cpp` since it also can cover filter cases?